modules/tensor_mechanics/src/materials/ComputeFiniteStrain.C

```diff
@@ -36,6 +36,8 @@ ComputeFiniteStrain::validParams()
 ComputeFiniteStrain::ComputeFiniteStrain(const InputParameters & parameters)
   : ComputeIncrementalStrainBase(parameters),
     _Fhat(_fe_problem.getMaxQps()),
+    _def_grad_mid(declareProperty<RankTwoTensor>(_base_name + "def_grad_mid")),
+    _f_bar(declareProperty<RankTwoTensor>(_base_name + "f_bar")),
```

Comment:
we should only declare these properties if the user selected H-W

Reply:
Make those pointers and set them to `nullptr` if not using HW

Reply:
:+1: